### DISCUSSION TITLE
Welcome to Fornjot!

### DISCUSSION THREAD

**Discussion by hannobraun** [hannobraun/fornjot#34] [Announcements]
Hey there!

Welcome to the Fornjot project. I'm Hanno, creator and maintainer of Fornjot. I'm a software developer who's interested in CAD, and I've been experimenting with various approaches to CAD software for a few years now. I think there's a real gap in the CAD market, which is why I'm putting a lot of effort into developing and promoting Fornjot. If you want to help out, please feel free to contribute to the project, or [consider supporting me financially](https://github.com/sponsors/hannobraun).

I'm also interested in the wider field of making physical things. [I have a website](https://hanno.braun-odw.eu/) where I write about CAD, open source, making, and small-scale manufacturing.

Now I'm wondering, who are you? What brings you here? Please consider introducing yourself by commenting below. I'd love to get to know you a bit better!

**Comment by xpe** [comment]
I'm here because of a long-running interest in building things: software, 3D models, woodworking, and more. While I like building some 3D models "by hand", I have some use cases where programmatic generation is the only feasible approach. The combination of building 3D models using the Rust language is a powerful combination.
